BSP/Inc/BSP_Contactor.h

```diff
@@ -4,6 +4,24 @@
 #define BSP_CONTACTOR_H
 
 #include "common.h"
+#include "stm32f4xx_gpio.h"
+#include "stm32f4xx_rcc.h"
+
+typedef uint8_t CONT_CHOICE;
+static const CONT_CHOICE ARRAY_CONTACTOR = (1 << 0);
+static const CONT_CHOICE HVHIGH_CONTACTOR = (1 << 1);
+static const CONT_CHOICE HVLOW_CONTACTOR = (1 << 2);
+static const CONT_CHOICE ALL_CONTACTORS = ARRAY_CONTACTOR | HVHIGH_CONTACTOR | HVLOW_CONTACTOR;
+
+// Contactor 1
+#define C1_PORT		GPIOB
+#define C1_PERIPH 	RCC_AHB1Periph_GPIOB
+// Contactor 2
+#define C2_PORT		GPIOA
+#define C2_PERIPH 	RCC_AHB1Periph_GPIOA
+// Contactor 3
+#define C3_PORT		GPIOC
+#define C3_PERIPH 	RCC_AHB1Periph_GPIOC
```

Comment:
I feel like this gets redundant, you are just redefining an already-defined peripheral but only using it once inside the file (not really used externally. Just for good coding practice, I don't really think its necessary. If you or @tiandahuang have any other opinions then maybe I'm not considering something.